File: KSXML.Server/KSXML/KSXML/KSXmlRecordSet.cs
using System.Collections.Generic;
using System.Text;

namespace KSXML
{
    public class KSXmlRecordSet
    {
        private readonly List<KSXmlRecord> _recordList = new List<KSXmlRecord>();

        public void AddRecord(KSXmlRecord xmlRecord)
        {
            _recordList.Add(xmlRecord);
        }

        public string ToXml()
        {
            var sbuilder = new StringBuilder();

            sbuilder.Append("<ksxmlrecordset>");
            _recordList.ForEach(xmlRecord => sbuilder.Append(xmlRecord.ToXml()));
            sbuilder.Append("</ksxmlrecordset>");

            return sbuilder.ToString();
        }
    }
}

File: KSXML.Client/MyPlayList/src/com/ksxml/source/KSXmlRecordSet.java
package com.ksxml.source;

import java.util.ArrayList;
import java.util.List;

public class KSXmlRecordSet {

	private List<KSXmlRecord> _recordList = new ArrayList<KSXmlRecord>();
	
	public KSXmlRecordSet() {
		
	}
	
	public void addRecord(KSXmlRecord xmlRecord) {
		this._recordList.add(xmlRecord);
	}
	
	public List<KSXmlRecord> getRecordList() {
		return this._recordList;
	}
}
